### PR TITLE
Add details for reporting issues

### PR DESCRIPTION
**Pull request by fabaff**
**Description:**
Add details for reporting issues.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#12989

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/help/reporting_issues.markdown [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+Please add the link to the documention of the component/platform in question. E.g.,
+
+- issue with the `random` sensor: [https://www.home-assistant.io/components/sensor.random/](/components/sensor.random/)
+- issue with the `hue` component: [https://www.home-assistant.io/components/hue/](/components/hue/)
+
+### {% linkable_title Description of problem %}
+
+Provide a summary of your issue and tell us what's wrong.
+
+There are components and platform which requires addition steps to get them working. Please describe the steps you took and the ones to reproduce the issue if needed.
```
Comment (MartinHjelmare) [inline source/help/reporting_issues.markdown]: ... which require additional...

Comment (MartinHjelmare) [reply source/help/reporting_issues.markdown]: Probably:
... which require additional steps to get working.

File: source/help/reporting_issues.markdown [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+```yaml
+sensor:
+  - platform: random
+```
+
+Make sure that you don't post your username, password, API key, access token or other [secrets](/docs/configuration/secrets/).
+
+### {% linkable_title Traceback (if applicable) %}
+
+If things go wrong there will be a so-called trackback or an error message in other works in your log. Please include this. It starts with **Traceback** and can contain informations where the error was triggered in the code.
```
Comment (MartinHjelmare) [inline source/help/reporting_issues.markdown]: ... traceback or an error message in other words in your log.

**Review by MartinHjelmare — COMMENTED** [review]
Nice!

File: source/help/reporting_issues.markdown (hunk 1 above)
Comment (fabaff) [reply source/help/reporting_issues.markdown]: Thanks

File: source/help/reporting_issues.markdown (hunk 2 above)
Comment (fabaff) [reply source/help/reporting_issues.markdown]: Thanks

**Pull request merged**